public/locales/en/modules/usenet.json

```diff
@@ -31,7 +31,8 @@
       "title": "Error",
       "message": "An error has occurred"
     },
-    "paused": "Paused"
+    "paused": "Paused",
+    "notImplemented": "NOT IMPLEMENTED"
```

Comment:
So no need to translate if it's not there lol. Just adds work for the ~25 languages we translate in for no reason.